src/components/TopNavBar/NavContent.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import "./NavStyles.css"
+import Hamburger from "./Hamburger";
+
+export default function NavContent() {
+    return (
+        <div className="navContent">
+            <div id="hamBox">
+                <div id="hamButton">
+                    <Hamburger />
+                </div>
+                <ul id="hamNavList">
+                    <li className="navItem hamNavItem"><a href="../Home" className="navItemLink">placeHolder1</a></li>
+                    <li className="navItem hamNavItem"><a href="../Home" className="navItemLink">placeHolder2</a></li>
+                    <li className="navItem hamNavItem"><a href="../Home" className="navItemLink">placeHolder3</a></li>
+                    <li className="navItem hamNavItem"><a href="../Home" className="navItemLink">placeHolder4</a></li>
```

Comment:
I think the figma design has the nav links. Here is the link: https://www.figma.com/file/hnVa7Jpi6vMFDIVyfTKyR5/PORCH-Initial-Design?node-id=88%3A2

There is a little tab on the left of the figma that says pages, and the prototype page has the designs for that.